microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,602 @@
+
+1. Click **Build** to generate the IL2CPP build. This will open a file dialog to select the folder for the build files. We recommend you create a new folder (such as **Build**) under your project directory for this purpose. Once you select your build folder in the file dialog, Unity will begin generating the build. This can take a few minutes.
+
+1. When this operation completes, Unity should launch a new File Explorer with the output folder selected. Otherwise, navigate to the output folder manually.
+
+1. Under your build folder, you will find a Visual Studio solution file (.sln). Double-click this file to open it in Visual Studio.
+
+1. In Visual Studio, select the **Solution Configuration** (such as **Release**). Then set the **Platform** to **ARM64**:
+
+![The Solution Configuration and Platform settings dropdown lists in Visual Studio's toolbar](./hololens2-images/visualstudio_buildconfig.png)
+
+1. Make sure **Developer Mode** is enabled on your PC and on your HoloLens 2 device. See [Enabling Developer Mode](/windows/mixed-reality/develop/advanced-concepts/using-visual-studio?tabs=hl2#enabling-developer-mode) in _Using Visual Studio to deploy and debug_ in the Mixed Reality documentation.
+
+1. Click the **deployment target** drop-down and then do one of the following:
+
+    * If you're building and deploying via Wi-Fi, select **Remote Machine** as the deployment target:
+
+      !['Remote Machine' selected in the 'deployment target' dropdown menu in Visual Studio's toolbar](./hololens2-images/visualstudio_select_remote.png)
+
+    * If you're building and deploying via USB, select **Device** as the deployment target:
+
+      !['Device' selected in the 'deployment target' dropdown menu in Visual Studio's toolbar](./hololens2-images/visualstudio_select_device.png)
+
+
+<!-- ------------------------------ -->
+#### Remote Connection
+
+To set up your remote connection:
+
+1. On the menu bar, select **Project > Properties > Configuration Properties > Debugging**.
+
+1. Click the **Debugger to launch** drop-down and then select **Remote Machine**.
+
+1. In the **Machine Name** field, enter the IP address of your device.
+    * You can find the IP address on your HoloLens 2 under **Settings > Network & Internet > Advanced Options**.
+    * We recommend that you manually enter your IP address rather than depend on the "Auto Detected" feature:
+
+    ![The Debugging settings in Project Settings dialog with Machine Name set](./hololens2-images/visualstudio_remote_ip.png)
+
+1. Set the **Authentication Mode** to **Universal (Unencrypted protocol)**.
+
+1. Build, deploy, and debug your app based on your needs:
+    * To build, deploy, and start debugging, select **Debug > Start debugging**.
+    * To build and deploy without debugging, select **Build > Deploy Solution**.
+
+1. The first time you deploy an app to your HoloLens 2 from your PC, you'll be prompted for a PIN. Follow the Pairing your device instructions below.
```

Comment:
@michaelfarnsworth - where's "Pairing your device" instructions?
locally changed to:
```
Follow the **Pairing your device**<!--todo--> instructions below.
```